examples/wrapper-components/vue-javascript/src/components/Tabs.vue

```diff
@@ -3,22 +3,60 @@
 <template>
   <div>
     <h2>Tabs</h2>
-    <ifx-tabs orientation="horizontal">
-      <ifx-tab header="tab 01">
-        This is tab 1
+    <ifx-tabs @ifxTabChange="handleChange" ref="tabsElement" orientation="horizontal">
+      <ifx-tab header="a" :disabled="false">
+        Content for Tab #1. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+        Praesent volutpat, ligula eu aliquam bibendum, orci nisl cursus ipsum, nec
+        egestas odio sapien eget neque. Lorem ipsum dolor sit amet, consectetur
+        adipiscing elit. Praesent volutpat, ligula eu aliquam bibendum, orci nisl
+        cursus ipsum, nec egestas odio sapien eget neque. Lorem ipsum dolor sit
+        amet, consectetur adipiscing elit. Praesent volutpat, ligula eu aliquam
+        bibendum, orci nisl cursus ipsum, nec egestas odio sapien eget neque.
       </ifx-tab>
-      <ifx-tab header="tab 02">
-        tabbbbbb 2
+      <ifx-tab header="b" :disabled="true">
+        Content for Tab #2. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+        Praesent volutpat, ligula eu aliquam bibendum, orci nisl cursus ipsum, nec
+        egestas odio sapien eget neque. Lorem ipsum dolor sit amet, consectetur
+        adipiscing elit. Praesent volutpat, ligula eu aliquam bibendum, orci nisl
+        cursus ipsum, nec egestas odio sapien eget neque. Lorem ipsum dolor sit
+        amet, consectetur adipiscing elit. Praesent volutpat, ligula eu aliquam
+        bibendum, orci nisl cursus ipsum, nec egestas odio sapien eget neque.
       </ifx-tab>
-      <ifx-tab header="tab 03">
-        Tab 3
+      <ifx-tab header="c" :disabled="false">
+        Content for Tab #3. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+        Praesent volutpat, ligula eu aliquam bibendum, orci nisl cursus ipsum, nec
+        egestas odio sapien eget neque. Lorem ipsum dolor sit amet, consectetur
+        adipiscing elit. Praesent volutpat, ligula eu aliquam bibendum, orci nisl
+        cursus ipsum, nec egestas odio sapien eget neque. Lorem ipsum dolor sit
+        amet, consectetur adipiscing elit. Praesent volutpat, ligula eu aliquam
+        bibendum, orci nisl cursus ipsum, nec egestas odio sapien eget neque.
       </ifx-tab>
     </ifx-tabs>
   </div>
 </template>
 
 <script setup>
+import { onMounted, ref } from 'vue'
 
+const tabsElement = ref(null);
+
+onMounted(() => {
+  setTab();
+  setInterval(setTab, 20000);
+})
+
+function handleChange(event) {
+  console.log("emitting active tab index: ", event.detail)
+};
+
+function setTab() {
+  const next = Math.floor(Math.random() * (3));
+  console.log("set next active tab: ", next)
+  if (tabsElement.value) {
+    tabsElement.value.$el.setActiveTab(next);
```

Comment:
Like in React, there is an error here:
**Cannot read properties of undefined (reading 'setActiveTab')**